SPECS/kernel-alt.spec

```diff
@@ -896,6 +915,15 @@ fi
 %{python2_sitearch}/*
 
 %changelog
+* Thu Apr 01 2021 Rushikesh Jadhav <rushikesh7@gmail.com> - 4.19.154-2
+- Security (XSAs 367 and 371) and bugfix update
+- XSA-367: Linux: netback fails to honor grant mapping errors
+- XSA-371: Linux: blkback driver may leak persistent grants
+- Patches backported from linus kernel to fix event-related issues caused by XSA-332
+
+* Tue Mar 30 2021 Rushikesh Jadhav <rushikesh7@gmail.com> - 4.19.154-1
+- Update patch level to 4.19.154
+
```

Comment:
This should be only one changelog entry, since relelase 4.19.154-1 will never get built and released

Reply:
Ok, I have merged the change logs.